docs/ref/models/fields.txt

```diff
@@ -502,6 +502,15 @@ Any combination of these options will result in an error.
     ``True`` will cause the field to have ``editable=False`` and ``blank=True``
     set.
 
+.. note::
+    When using timezones, storing a ``DateField`` in the database is
+    ambiguous because it's incapable of storing timezone aware information, unlike ``DateTimeField``.
```

Comment:
I should note here that @chrisjluc adapted this fairly directly from what I wrote on the ticket. 

While I think what you say here is true in the broad sense, in the specific case of auto now (add) I think it's entirely accurate to say that storing a date is a lossy and ambiguous representation of the original data, where a datetime is precise. In the case of auto now (add) the original data is, by definition, a point in time. If you store that point in time, it is always possible to convert that to a date in whatever timezone you like. If you store the date, you have irretrievably lost information, and you cannot convert back.

You may be right that this doc change could be improved though; I'm on my phone right now and would need to take a deeper look at the context to make a proposal.